features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/model/TimelineItem.kt

```diff
@@ -69,6 +70,10 @@ sealed interface TimelineItem {
         val showSenderInformation = groupPosition.isNew() && !isMine
 
         val safeSenderName: String = senderDisplayName ?: senderId.value
+
+        val failedToSend: Boolean = sendState is EventSendState.SendingFailed
+
+        val isTextMessage: Boolean = content is TimelineItemTextBasedContent
```

Comment:
Could we use this to replace the `isMediaItem` check here: https://github.com/vector-im/element-x-android/blob/bb6ceb4541463922e46cbe995e99de49014596e4/features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/TimelineItemEventRow.kt#L231-L233

or are they not quite the same?

Reply:
Not exactly, that isMediaItem is used to have different paddings and spacing in the message bubbles since their previews leave no borders. If we used `!isTextMessage` here instead, we'd have weird padding and spacing values for redacted, undecrypted or unknown events too.